docs/source/install.rst

```diff
@@ -98,47 +98,71 @@ Specific functionality in Dask may require additional optional dependencies.
-
-\* Note that ``toolz`` is a mandatory dependency but it can be transparently replaced with
-``cytoolz``.
-
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| Dependency      | Version     | Description                                                                                            |
++=================+=============+========================================================================================================+
+| bokeh           | >=2.4.2     | Visualizing dask diagnostics                                                                           |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| cachey          | >=0.1.1     | Use caching for computation                                                                            |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| cityhash        | >=0.2.4     | Faster hashing of arrays                                                                               |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| crick           | >=0.0.3     | Use ``tdigest`` internal method for percentile computation                                             |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| cytoolz         | >=0.10.1    | Faster cythonized implementation of internal iterators, functions, and dictionaries                    |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| dask-ml         | >=1.4.0     | Common machine learning functions scaled with Dask                                                     |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| fastavro        | >=0.22.6    | Storing and reading data from Apache Avro files                                                        |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| fastparquet     | >=0.8.2     | Storing and reading data from Apache Parquet files                                                     |
++-----------------+-------------+--------------------------------------------------------------------------------------------------------+
+| gcsfs           | >=0.4.0     | Storing and reading data located in Google Cloud Storage                                               |
```

Comment:
Worth noting here that `gcsfs` doesn't seem to be getting tested anywhere, so this min version is likely outdated - might be worth it to add some tests in the medium term, but I can see if we can bump this version based on our new `fsspec` min